### DISCUSSION TITLE
error=failed to build for darwin_amd64: # github.com/rjeczalik/notify

### DISCUSSION THREAD

**Discussion by kaihendry** [goreleaser/goreleaser#1986] [Q&A] [answered]
I want to cross compile for MacOS, but I am running into this issue:
https://github.com/rjeczalik/notify/issues/177

Here's my source: https://github.com/kaihendry/yyyymmdd/

Any ideas how to get around this? Perhaps a CI (GH Action) solution?

**Comment by caarlos0** [comment, marked as answer]
There is a cookbook linked in the docs for building with CGO. https://goreleaser.com/cookbooks/cgo-and-crosscompiling/

**Reply by kaihendry** [reply]
Thanks though I've been studying https://github.com/troian/golang-cross-example/blob/master/.goreleaser.yaml and I don't understand how the Docker image **troian/golang-cross** comes in. 

Do I have to create a sysroot folder and check it in via the https://github.com/troian/golang-cross-example/blob/master/Makefile?
